project/app/main.py

```diff
@@ -3,7 +3,7 @@
 import uvicorn
 from fastapi_utils.tasks import repeat_every
 
-from app.api import predict, viz, getdata, ping
+from app.api import predict, viz, getdata
```

Comment:
Was ping used, and if so is it still utilized?

Reply:
If 'predict' and 'viz' are still used here, this is scaffolding code and should be replaced with production code.